src/libs/actions/IOU.ts

```diff
@@ -7190,6 +7190,8 @@ function submitReport(expenseReport: OnyxTypes.Report) {
     const adminAccountID = policy?.role === CONST.POLICY.ROLE.ADMIN ? currentUserPersonalDetails?.accountID : undefined;
     const optimisticSubmittedReportAction = ReportUtils.buildOptimisticSubmittedReportAction(expenseReport?.total ?? 0, expenseReport.currency ?? '', expenseReport.reportID, adminAccountID);
     const optimisticNextStep = NextStepUtils.buildNextStep(expenseReport, isSubmitAndClosePolicy ? CONST.REPORT.STATUS_NUM.CLOSED : CONST.REPORT.STATUS_NUM.SUBMITTED);
+    const approvalChain = ReportUtils.getApprovalChain(PolicyUtils.getPolicy(expenseReport?.policyID), expenseReport);
+    const managerID = approvalChain.length ? Number(approvalChain.at(0)) : undefined;
```

Comment:
Please update the PR again, approvalChain is string array

<img width="348" alt="Screenshot 2024-12-02 at 16 58 52" src="https://github.com/user-attachments/assets/c71d8a2c-9997-4d58-a9ed-8e71883cca73">